### PR TITLE
Added support for docker compose 🛳

### PR DESCRIPTION
**Pull request by exelash**
Added support for Docker Compose, this allows for an easier time self-hosting.

### CODE REVIEW

**Comment by exelash** [comment]
*If accepted I will be willing to update the documentation*

**Comment by marinofranz** [comment]
Leaving DATABASE_URL blank will leave Prisma non-functional as far as I know.

**Comment by exelash** [comment]
> 

There's a fall back in `docker-compose.yml`, It automatically sets the environment variable.

**Comment by exelash** [comment]
> Leaving DATABASE_URL blank will leave Prisma non-functional as far as I know.

There's a fall back in `docker-compose.yml`, It automatically sets the environment variable.

**Review by marinofranz — COMMENTED** [review]
Have you tested this in a local environment? I'll be able to test it shortly.

**Comment by exelash** [comment]
> Have you tested this in a local environment? I'll be able to test it shortly.

Yes, tested on both macOS 12.3.1 and ubuntu 22.04.

File: .env.example [hunk 1]
```diff
@@ -6,8 +6,8 @@ SESSION_SECRET= your-super-secret-32-char-key # Generate one here --> https://bi
 DATABASE_URL= postgresql://USER:PASSWORD@HOST:PORT/DATABASE # Levae this blank if you are self hosting with docker compose
```
Comment (marinofranz) [inline .env.example:6]: Correct spelling errors:
(levae -> leave)
(docker compose -> Docker Compose)

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+############
+# Database - Only required if you are using postgres with docker compose
+# *Change password in production*
+############
+
+POSTGRES_PASSWORD= supersecretpassword
+POSTGRES_USER= postgres
+POSTGRES_DB= tovy
+POSTGRES_PORT= 5432
```
Comment (marinofranz) [inline .env.example:16]: Remove whitespace before the values.

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,17 @@
+############
+# Tovy
+############
+
+SESSION_SECRET= your-super-secret-32-char-key # Generate one here --> https://bitwarden.com/password-generator/
+DATABASE_URL= postgresql://USER:PASSWORD@HOST:PORT/DATABASE # Levae this blank if you are self hosting with docker compose
```
Comment (marinofranz) [inline .env.example:6]: Remove whitespace before the values.

**Review by marinofranz — CHANGES_REQUESTED** [review]
Can you please fix these issues? Re-request a review once that's finished.

**Comment by marinofranz** [comment]
I will test on my machine when I get the opportunity and then merge this. Thanks for the contribution! ❤️ 🎉

**Comment by exelash** [comment]
No problem 😊

**Pull request merged**